docs/concepts/models.md

```diff
@@ -1419,99 +1469,11 @@ print(error_locations)
 #> [('a',), ('b',), ('c',), ('d',), ('e',)]
 ```
 
-## Required fields
-
-To declare a field as required, you may declare it using an annotation, or an annotation in combination with a
-[`Field`][pydantic.Field] function (without specifying any `default` or `default_factory` argument).
-
-```python
-from pydantic import BaseModel, Field
-
-
-class Model(BaseModel):
-    a: int
-    b: int = Field(alias='B')
-    c: int = Field(..., alias='C')
-```
-
-Here `a`, `b` and `c` are all required. The field `c` uses the [ellipsis][Ellipsis] as a default argument,
-emphasizing on the fact that it is required. However, the usage of the [ellipsis][Ellipsis] is discouraged
-as it doesn't play well with type checkers.
```

Comment:
I think the verbosity of this section is helpful for more beginner users - could we keep some of this in the `fields.md` page?

Reply:
The very beginning of the `models.md` page and [the welcome page](https://docs.pydantic.dev/latest/#pydantic-examples) already explains this, so at this point I expect the user to already know about this.

At the beginning of the `fields.md` page, a note is present (see https://github.com/pydantic/pydantic/pull/11029#discussion_r1866636664).